### DISCUSSION TITLE
An option to stop the tabs on the bottom from automatically opening / switching

### DISCUSSION THREAD

**Discussion by The2AndOnly** [godotengine/godot-proposals#11087] [Editor]
I like to work with the bottom tab closed, since it takes a lot of space. But there is an issue that has bothered me ever since I began working with Godot. It's that the tab will automatically open whenever I click on certain nodes in the scene tree.

For example, if I click on an AnimatedSprite2D then it will automatically open / switch to the SpriteFrames tab.
And if I click on an AnimationPLayer then it will automatically open / switch to the Animation tab.

The really annoying part is that it stays opened once I click off of the node, so I'm forced to go down and close it. It's really annoying for me and I constantly have to go down and close the tab whenever I click on a node in order to manage it's properties.

I propose that there would be an editor setting (a checkbox) which can toggle this behavior from happening.

**Comment by tetrapod00** [comment]
Related: https://github.com/godotengine/godot/pull/98074 solves the opposite problem, allowing you to keep a bottom panel open even when it would normall close. It's possible that the same pin icon could lock the bottom panel *closed*, instead of open. That might have to be a separate proposal and PR after the PR is merged, though.

**Reply by The2AndOnly** [reply]
This didn't solve the issue, I still have to deal with the panel opening.
I've just gotten used to it at this point.

**Comment by redfacedev** [comment]
bump

**Comment by The2AndOnly** [comment]
I got an email that someone said pinning the panel in 4.6 will prevent the panel from opening again, but I'm using v4.6.stable.official [89cea1439] and that's not the case when I tried it. So as of 4.6, this issue is still unresolved.

I do wanna mention that sometime since when I made this issue, an update was added that will close the panel again once you click off of the node. So that definitely makes it much less annoying than when this issue was made, but there's still the problem of the panel opening in the first place.

**Reply by redfacedev** [reply]
For me, in 4.6 Stable, pinning the panel and then closing it keeps it closed when running the project. However it reopens when you click on a node such as a Polygon2D. I am currently working on a pull request to fix that.

**Comment by Repiteo** [comment]
Implemented by godotengine/godot#115978!
